.circleci/config.yml

```diff
@@ -0,0 +1,372 @@
+version: 2.1
```

Comment:
Circle has been decommissioned. Incorporate any changes into the gihub action for CI

Reply:
This doesn't seem resolved. Shouldn't this file go away if we've moved to GitHub actions?

Reply:
This hasn't been resolved and Circle config is back in again 😢